experimenter/experimenter/experiments/constants.py

```diff
@@ -184,6 +184,18 @@ class ApplicationConfig:
     is_web=True,
 )
 
+APPLICATION_CONFIG_FXA_WEB = ApplicationConfig(
+    name="Fxa Web",
```

Comment:
This should probably be "Firefox Accounts" or something

Reply:
you know originally I had firefox accounts and then I changed it to fxa, let me put that back